pkg/policy/execute.go

```diff
@@ -124,7 +124,7 @@ func (e *Executor) with(policy string, args ...interface{}) *Executor {
 // Execute executes given policy and the related sub queries/views.
 func (e *Executor) Execute(ctx context.Context, req *ExecuteRequest, policy *Policy) (*ExecutionResult, error) {
 	if !policy.HasChecks() {
-		return nil, fmt.Errorf("no checks or policies to execute")
+		e.log.Warn("no checks or policies to execute")
```

Comment:
Would set result := ExecutionResult{PolicyName: req.Policy.Name, Passed: true, Results: make([]*QueryResult, 0)} and the if below do return result. 

This define is instead of line 136